scripts/src/admin/wrapper.html

```diff
@@ -333,15 +333,27 @@ <h1><a href="/">Marketplace Admin</a></h1>
 				const appId = editor.frame.parentNode.dataset.appId;
 				try {
 					json = editor.get();
+					if (

```

Comment:
again, this format isn't fun to read, extract the function into a variable, then use it and do it as a one liner.
and as it looks like the same function, just save it in a place where both can use it